caddy/README.md

```diff
@@ -10,16 +10,30 @@ Shadowsocks backend.
 
 ## Usage
 
-From this directory, build and run a custom binary with `xcaddy`:
+From this directory, build and run a custom Caddy binary with this Go module plugged in using `xcaddy`:
 
 ```sh
-xcaddy run --config config_example.json --watch
+xcaddy build \
```

Comment:
This is complicated
Perhaps we should to provide our own main that pulls all those modules.